instaclone/src/components/PostContainer.js

```diff
@@ -3,10 +3,9 @@ import "./head.css"
 import PropTypes from 'prop-types';
 import Comment from "./Comment";
 const PostContainer = (props)=>{
```

Comment:
I would like to see some space between this line of code and your imports. This is convention and helps other developers quickly understand what's happening inside your code.

right now it looks like your component declaration is an import unless you look twice.
```suggestion
const PostContainer = (props)=>{
```